pyincore/analyses/pipelinedamagerepairrate/pipelinedamagerepairrate.py

```diff
@@ -225,78 +226,68 @@ def pipeline_damage_analysis_bulk_input(self, pipelines, hazard_type,
-                            elif 'pipelinesc' in pipeline['properties']:
-                                damage_result['pipeclass'] = pipeline['properties']['pipelinesc']
-                            else:
-                                damage_result['pipeclass'] = ""
-                        else:
-                            raise ValueError("One of the fragilities is in deprecated format. This should not happen. "
-                                             "If you are seeing this please report the issue.")
+            hazard_vals = AnalysisUtil.update_precision_of_lists(hazard_resp[i]["hazardValues"])
+            demand_types = hazard_resp[i]["demands"]
+            demand_units = hazard_resp[i]["units"]
 
-            else:
-                raise ValueError("One of the fragilities is in deprecated format. This should not happen. If you are "
-                                 "seeing this please report the issue.")
+            hval_dict = dict()
+            for j, d in enumerate(fragility_set.demand_types):
+                hval_dict[d] = hazard_vals[j]
+
+            if not AnalysisUtil.do_hazard_values_have_errors(hazard_resp[i]["hazardValues"]):
+                pipeline_args = fragility_set.construct_expression_args_from_inventory(pipeline)
+                pgv_repairs = \
+                    fragility_curve.calculate_limit_state_probability(
+                        hval_dict, fragility_set.curve_parameters, **pipeline_args)
+                # Convert PGV repairs to SI units
+                pgv_repairs = PipelineUtil.convert_result_unit(fragility_curve.return_type["unit"], pgv_repairs)
+
+                length = PipelineUtil.get_pipe_length(pipeline)
+
+                # Number of PGV repairs
+                num_pgv_repairs = pgv_repairs * length
+
+                # Check if liquefaction is applicable
+                if use_liquefaction is True \
+                        and fragility_sets_liq is not None \
+                        and geology_dataset_id is not None \
+                        and liquefaction_resp is not None:
+                    fragility_set_liq = fragility_sets_liq[pipeline["id"]]
+
+                    # TODO assume there is only one curve
+                    liq_fragility_curve = fragility_set_liq.fragility_curves[0]
+
+                    liq_hazard_vals = AnalysisUtil.update_precision_of_lists(liquefaction_resp[i]["pgdValues"])
+                    liq_demand_types = liquefaction_resp[i]["demands"]
+                    liq_demand_units = liquefaction_resp[i]["units"]
+                    liquefaction_prob = liquefaction_resp[i]['liqProbability']
+                    liq_hval_dict = dict()
+                    for j, d in enumerate(liquefaction_resp[i]["demands"]):
+                        liq_hval_dict[d] = liq_hazard_vals[j]
+
+                    # !important! removing the liqProbability and passing in the "diameter"
```

Comment:
what does this mean?

Reply:
@longshuicy or @vnarah2 do either of you know what this comment means? I think one of you wrote the comment. My removing the now unnecessary dfr3curve check makes it look like this is a bigger change than it really is. I didn't add that comment. Any ideas? This might just be referring to the fact that after getting the liquefaction, we are only making a dictionary of pgd values and excluding the liqProbability.